config/testing.json

```diff
@@ -23,6 +23,13 @@
     "base_logger": "testmecha",
     "log_file": "logs/unit_tests.log"
   },
+  "database": {
+    "host": "localhost",
+    "port": 5432,
+    "dbname": "circle_test",
+    "username": "root",
+    "password": "mecha"
+  },
```

Comment:
please also append this region to the template config file, don't want that getting out of date.

Reply:
Added in ade5999.